### DISCUSSION TITLE
UE-Explorer-1.4.0 (Canary)

### DISCUSSION THREAD

**Discussion by EliotVU** [UE-Explorer/UE-Explorer#24] [Announcements]
A canary release for of version 1.4.

With this update comes Docking! A major UI overhaul in the works to displace the static panels, bugs included!
This update also uses the latest UELib which brings support for Unreal Engine 4 .uasset files (earlier versions only) and support for Duke Nukem Forever (2011).

<hr /><em>This discussion was created from the release <a href='https://github.com/UE-Explorer/UE-Explorer/releases/tag/Release-DNF_Preview'>UE-Explorer-1.4.0 (preview)</a>.</em>

**Comment by Shtoyan** [comment]
Thanks, works like charm!
